packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -15,6 +15,19 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
   - `attributes` property on the `IZoweUSSTreeNode` interface - use the `getAttributes` and `setAttributes` functions instead.
 - **Breaking:** Added return type of `Promise<void>` to `MainframeInteractions.ICommon.logout`. [#2783](https://github.com/zowe/vscode-extension-for-zowe/pull/2783).
 - Grouped Common methods into Singleton classes. [#2109](https://github.com/zowe/zowe-explorer-vscode/issues/2109)
+- **Breaking:** Removed deprecated methods: [#2238](https://github.com/zowe/zowe-explorer-vscode/issues/2238)
```

Comment:
Could we add this Breaking changes list to the branch's ./docs/early-access/v3/extenders.md file also please?

Reply:
I think going forward we want to document changes in the Box document and publish them on Zowe.org, but since that's quite new and users may be expecting to find the list in Extenders.md, I've updated it too for now 👍